src/entity/user/User.ts

```diff
@@ -131,6 +131,9 @@ export class User {
   @Column({ default: false })
   devcardEligible: boolean;
 
+  @Column({ default: true })
```

Comment:
For some reason thought we said flag but it's fine.

Reply:
We decided a column so we can easily ignore cdc for this

Reply:
Ah either way is easy, but it's ok :)